### PR TITLE
Support nil in ElementBuilder

### PR DESCRIPTION
**Pull request by kylebshr**
Support `nil` in all cases in `ElementBuilder` (and any other such specializations of `Builder`) by disfavoring the optional build expression function in `Builder`.

### CODE REVIEW

File: BlueprintUI/Sources/Layout/Builder.swift [hunk 1]
```diff
@@ -23,6 +23,10 @@ public struct Builder<Child> {
         [child]
     }
 
+    /// This function is disfavored in case a builder wants to offer additional `buildExpression` functions to support
```
Comment (kyleve) [inline BlueprintUI/Sources/Layout/Builder.swift:26]: TIL!

Comment (kyleve) [reply BlueprintUI/Sources/Layout/Builder.swift:26]: (Also ugh, we have like 5 copies of these result builders. I wonder if we should put these into a pod somewhere and use them in market, blueprint, listable at some point...)

**Review by kyleve — APPROVED** [review]
Nice find!

**Comment by ldstreet** [comment]
Awesome!

**Review by watt — APPROVED** [review]
- [ ] changelog

**Comment by kylebshr** [comment]
The changelog has an entry that captures this due to another change (Added support for optionals in builders without unwrapping via `if let`.)

**Pull request merged**